update/factory/exportHandlerFactory.go

```diff
@@ -291,11 +294,19 @@ func createFinalExportStorage(storageConfig config.StorageConfig, folder string)
 	return accountsTrieStorage, nil
 }
 
-func (e *exportHandlerFactory) setWhiteListHandlerToInterceptors() {
+func (e *exportHandlerFactory) setWhiteListHandlerToInterceptors() error {
+	var err error
+
 	e.interceptorsContainer.Iterate(func(key string, interceptor process.Interceptor) bool {
-		interceptor.SetIsDataForCurrentShardVerifier(e.whiteListHandler)
+		errFound := interceptor.SetIsDataForCurrentShardVerifier(e.whiteListHandler)
+		if errFound != nil {
+			err = errFound
+			return false
```

Comment:
why stopping here? maybe other interceptors succeed the setting of the whiteListHandler

Reply:
because if not stopping, the exporthandler might not work later. no need to start the export if not all the interceptors accept the whitelister.

Reply:
ok